modules/getting-started/pages/sync-api-apisync.adoc

```diff
@@ -1,21 +0,0 @@
-= To Sync an API
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
-:noindex:
-
-When you update a RAML file from Anypoint Studio, you can synchronize those changes to your API version hosted in Anypoint Platform. Assuming you created the placeholder API described in the previous sections, make some small change to the API. Navigate to your `placeholder.raml` file in the Package Explorer, and double-click to open the file in your RAML Editor. For example, edit the RAML to change a description slightly.
-
-== Synchronize your Updated RAML File With Anypoint Platform
-
-Notice that after making the change, the API Sync view and Upload become available for uploading changes:
```

Comment:
- Is this one or two things: "the API Sync view and Upload"?
Or is "Upload" a button or something ?

- "become available for uploading changes" > "are enabled so that you can upload it/the change/your change"

Reply:
great comments -- not taking action as we're deleting this topic